docs/using-the-jdbc-driver/using-plugins/UsingTheIamAuthenticationPlugin.md

```diff
@@ -24,14 +24,13 @@ IAM database authentication use is limited to certain database engines. For more
-| `iamRegion`                               | String  |    No    | This property will override the default region that is used to generate the IAM token. The default region is parsed from the connection string.                                                                                                                                                                        | `us-east-2`   |
-| `iamExpiration`                           | Integer |    No    | This property will override the default expiration time that is assigned to the generated IAM token. The default expiration time is set to be 15 minutes.                                                                                                                                                              | `600`         |
-
+| Parameter        |  Value  | Required | Description                                                                                                                                                                                                                                                                                                            | Example Value                                       |
+|------------------|:-------:|:--------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
+| `wrapperPlugins` | String  |    No    | Set to `"iam"` to enable AWS IAM database authentication                                                                                                                                                                                                                                                               | `iam`                                               |
+| `iamDefaultPort` | String  |    No    | This property will override the default port that is used to generate the IAM token. The default port is determined based on the underlying driver protocol. For now, there is support for `jdbc:postgresql:` and `jdbc:mysql:`. Target drivers with different protocols will require users to provide a default port. | `1234`                                              |
+| `iamHost`        | String  |    No    | This property will override the default hostname that is used to generate the IAM token. The default hostname is derived from the connection string. This parameter is useful when using SSH tunneling is in use and the connection string uses `localhost` instead of an RDS endpoint to establish connections.       | `database.cluster-hash.us-east-1.rds.amazonaws.com` |
```

Comment:
I'm not sure I understand this sentence. Can we re-word it?

> This parameter is useful when using SSH tunneling is in use and the connection string uses localhost instead of an RDS endpoint to establish connections.